package.json

```diff
@@ -13,7 +13,8 @@
     "prepare": "husky install",
     "lint": "eslint --ext .js,.vue --ignore-path .gitignore --fix .",
     "lint:ci": "eslint --ext .js,.vue --ignore-path .gitignore .",
-    "format": "prettier . --ignore-path .gitignore --write",
+    "format": "prettier --ignore-path .gitignore --write .",
+    "pre-commit": "npx pretty-quick --staged && npm run lint:ci",
```

Comment:
You can do `npm run lint` here. The `:ci` variant was just for GitHub Actions (drops the --fix flag since GHA can't update the repo)

Reply:
Done!